staker/stateless_block_validator.go

```diff
@@ -428,8 +429,13 @@ func (v *StatelessBlockValidator) Start(ctx_in context.Context) error {
 			return fmt.Errorf("starting execution spawner: %w", err)
 		}
 	}
-	for _, spawner := range v.execSpawners {
+	for i, spawner := range v.execSpawners {
 		if err := spawner.Start(ctx_in); err != nil {
+			if u, parseErr := url.Parse(v.config.ValidationServerConfigs[i].URL); parseErr == nil {
+				if u.Scheme == "http" || u.Scheme == "https" {
```

Comment:
How about explicitly testing that it isn't `ws` or `wss`?  That way it would catch crazy things like `ftp`, etc.

Reply:
I agree, fixed it.